tests/unit/core/containers/TestServerHtml.js

```diff
@@ -26,6 +27,9 @@ describe('<ServerHtml />', () => {
     getState: () => ({ foo: 'bar' }),
   };
 
+  const amoCDN = 'https://test.cdn.net';
```

Comment:
This is a small request but it would be nicer to move this into the function for the test that uses it to prevent any scope surprises. In other words, no other test needs to access it so it shouldn't go at the top of the file. In the future, if more than one tests needs access than we can move it higher up in scope. The same thing applies to `const _config = ...`.